components/contextualhelp/metadata/contextualhelp.yml

```diff
@@ -0,0 +1,61 @@
+name: Contextual Help
+status: Verified
+SpectrumSiteSlug: https://spectrum.adobe.com/page/contextual-help/
+sections: 
+- name: Custom Properties API
+  description: |
+    This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a href="https://github.com/adobe/spectrum-css/tree/main/components/popover/metadata/mods.md">here</a>.
+examples:
+  - id: contextualhelp-variants
+    name: Info Icon
+    markup: |
+      <div class="spectrum-Examples">
+        <div class="spectrum-Examples-item">
+          <div class="spectrum-ContextualHelp">
+            <button class="spectrum-ActionButton spectrum-ActionButton--sizeXS spectrum-ContextualHelp-button">
```

Comment:
should we be using `spectrum-ActionButton--sizeS`  when scale is Large. 
![Screenshot 2023-03-21 at 12 08 40 PM](https://user-images.githubusercontent.com/63808889/226702225-aa5c159a-daf7-4830-b46b-933e7ce38475.png)

Reply:
The issue is just that I have to specify the class here for the docs. When someone is implementing it they should use the correct sizing. Do you think I should provide an example of both? I don't think I have a way to dynamically change that if someone changes the setting on the docs site to Large, unless you have an idea?

Reply:
Oh we're going to be using our Storybook examples on the docs site?

Reply:
Eventually - that's why we wrote the templates so scoped so we could eventually leverage them in other contexts.